features/networking/egress-ip.feature

```diff
@@ -433,11 +433,12 @@ Feature: Egress IP related features
     """
 
     # Create a pod
-    Given I obtain test data file "routing/web-server-1.yaml"
+    Given I obtain test data file "networking/externalip_pod.yaml"
     When I run the :create client command with:
-      | f | web-server-1.yaml |
+      | f | externalip_pod.yaml |
```

Comment:
@huiran0826 Why is the YAML file changed? It would be helpful if we add a comment describing the intent of PR.